fitwsarah_frontend/src/components/clientProfile/clientInvoices.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import './Invoices.css';
+import { useGetAccessToken } from "../authentication/authUtils";
+
+
+function ClientInvoices({userId}) {
+
+    const [invoices, setInvoices] = useState([]);
+    const [accessToken, setAccessToken] = useState(false);
+    const getAccessToken = useGetAccessToken();
+
+    useEffect(() => {
+        const fetchToken = async () => {
+            const token = await getAccessToken();
+            if (token) setAccessToken(token);
+        };
+
+        fetchToken();
+    }, [getAccessToken]);
+
+    useEffect(() => {
+        if (accessToken) {
+            getInvoicesByUserId(userId);
+        }
+    }, [accessToken]);
+
+    function getInvoicesByUserId(userId) {
+        fetch(`http://localhost:8080/api/v1/invoices/users/${userId}`, {
+            method: "GET",
+            headers: new Headers({
+                Authorization: "Bearer " + accessToken,
+                "Content-Type": "application/json"
+            })
+        })
+            .then((response) => {
+                if (!response.ok) {
+                    throw new Error('Network response was not ok ' + response.statusText);
```

Comment:
You forgot to change the url

Reply:
thx for letting me know, I just  added it .